assets/dist/js/toptalent.js

```diff
@@ -0,0 +1,140 @@
+document.addEventListener('DOMContentLoaded', () => {
+  // creating variables
+  const flow = document.querySelector('.homepage');
+  const talentHolder = document.createElement('section');
+  talentHolder.classList.add('toptalent');
+
+  // header to talent section
+  const talentHead = document.createElement('article');
+  talentHead.classList.add('toptalentheader');
+  flow.appendChild(talentHolder);
+
+  // parts of the header
+  const talentH2 = document.createElement('h2');
+  talentH2.classList.add('talentheader');
+  talentH2.textContent = 'Our Top Talents';
+
+  const talentImg = document.createElement('img');
+  talentImg.classList.add('line');
+  talentImg.src = 'assets/images/patterns/underline.png';
+  talentImg.setAttribute('alt', 'a line for the page');
+
+  talentHead.append(talentH2, talentImg);
+  talentHolder.appendChild(talentHead);
+
+  // end of title section
+  // start to lawyers section
+
+  const lawyersContainer = document.createElement('div');
+  lawyersContainer.classList.add('tcontainer');
+  talentHolder.append(lawyersContainer);
+
+  const talents = [
+    {
+      img: 'assets/images/act/lawyerp1.jpg',
+      header: 'Consultation',
```

Comment:
- [ ] please ensure that Each article has two columns, the first with the image at the left and the second with the title, subtitle, and description like in the original design
yours
![Screenshot (1006)](https://user-images.githubusercontent.com/65298048/234984879-b583a22f-5b49-44e0-97f5-0b947bd58dd6.png)
behance design
![Screenshot (1012)](https://user-images.githubusercontent.com/65298048/234984930-7b020f83-a99b-4208-a7ab-b228755dfea4.png)